src/sql/workbench/contrib/notebook/browser/notebookLinkHandler.ts

```diff
@@ -118,6 +118,20 @@ export class NotebookLinkHandler {
 		}
 	}
 
+	public getEncodedLinkUrl(linkUrl: string): string {
```

Comment:
These don't need to be a part of the class - just have them be module level functions